### ISSUE TITLE
Admin tables order

### ISSUE THREAD

**Issue by decabeza** [consul/consul#2931]
## Background

In `/views/admin/budget_investments/_investments.html.erb` on Admin panel, there is a select form to sort by some attributes of the table.

## What we need

Replace this select form to a link in each `<th>` to order by them.

## Notes

For te arrow icon you can use the existing one `<span class="icon-arrow-down">`.

The objective of this issue is replacing it, but it would be nice adds this feature in all tables of the admin panel. 😏 

## Screenshots

**[BEFORE]**
![before](https://user-images.githubusercontent.com/631897/46139064-dbc21480-c24d-11e8-9a4c-29a29336d931.png)

**[AFTER]**
![after](https://user-images.githubusercontent.com/631897/46139071-dd8bd800-c24d-11e8-8de3-d85587799eaf.png)

**Comment by matisnape** [comment]
I would like to take a shot at it, if possible :)

**Comment by javierm** [comment]
@matisnape Sure. Go ahead!

**Comment by javierm** [comment]
Hi @matisnape! How are you doing? Are you still working on this issue? If so, feel free to ask us if you need any help :wink:. Thank you very much! :pray:

**Comment by matisnape** [comment]
@javierm Yes, I'm planning to do that during holidays - I had some problems in setting up the database, but I've managed to start the server locally. Sorry for the delay :)

**Comment by matisnape** [comment]
@javierm Hey, just a question - according to the `scope :sort_by_title`, title column is sorted ascending and screenshot contains arrow down. Does that mean that I should implement the sorting both ways? What about other columns that have arrows on the screenshot? Should they be sortable too? :)

**Comment by javierm** [comment]
@matisnape Yes, implementing the sorting both ways would be great :smile:.

**Comment by javierm** [comment]
@matisnape I've left this issue unassigned in case other people want to contribute. If you still have time and would like to contribute, feel free to do so :wink:.

**Comment by matisnape** [comment]
Hacktoberfest has started so I think I will have another go :D

**Comment by lucialuzuriaga** [comment]
Closing this issue as won't do for now, reason being lack of capacity. Should this become a priority, or should anyone want to pick this up, then it can be reopened.
